corehq/apps/reports/views.py

```diff
@@ -1477,9 +1477,9 @@ def clear_report_caches(request, domain):
 def export_report(request, domain, export_hash, format):
     cache = get_redis_client()
 
-    if cache.exists(export_hash):
```

Comment:
are you running the latest version of code on this branch? because `get_redis_client` no longer returns the raw client but now just returns the cache object (see previous comment for why that's the case)